.github/workflows/deploy-daac-test.yml

```
@@ -1,10 +1,9 @@
-name: Deploy DAAC Stacks to AWS
+name: Deploy DAAC test to AWS
 
 on:
   push:
-    branches:
-      - main
-      - develop
+    tags:
+      - 'v*'
```

`develop` for the test environment, yes?
```suggestion
    branches:
      - develop
```

---

Oh woops!